meinberlin/apps/cms/blocks.py

```diff
@@ -27,7 +27,7 @@ class ProjectTeaserBlock(blocks.StructBlock):
         help_text=_("Will be shown instead of the project title if set."),
     )
     custom_teaser = blocks.RichTextBlock(
-        max_length=500,
+        max_length=250,
```

Comment:
Should we also truncate the project description in case no custom teaser is provided?